internal/tag/tag_test.go

```diff
@@ -49,7 +49,7 @@ func TestListTags(t *testing.T) {
 }
 
 func TestDeleteTags(t *testing.T) {
-	args := []string{common.TagName, common.TagName1, common.TagName2, common.TagName3, common.TagName4}
+	args := []string{common.TagName, "v1", "v2", "v3", "v4"}
```

Comment:
Whats the point of this change?

Reply:
It is to match up with the value of tags used while mocking (see line 67-70) in this file so that this test continue to pass.